src/main/g8/build.sbt

```diff
@@ -32,7 +32,8 @@ lazy val commonSettings = Def.settings(
   scalacOptions ++= Seq("-target:jvm-1.8",
                         "-deprecation",
                         "-feature",
-                        "-unchecked"),
+                        "-unchecked",
+                        "-Ymacro-annotations"),
```

Comment:
This only applies to scala 2.13; since the template supports 2.12 as well we need to add the macro paradise compiler plugin

Reply:
Added. I also had to bump the beam-flink dep. Can I get a re-review @regadas ?